monitoring/pom.xml

```diff
@@ -0,0 +1,76 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.cloud.monitoring.samples</groupId>
+  <artifactId>cloud-monitoring-samples</artifactId>
+  <packaging>jar</packaging>
+
+  <parent>
+    <artifactId>doc-samples</artifactId>
+    <groupId>com.google.cloud</groupId>
+    <version>1.0.0</version>
+    <relativePath>..</relativePath>
+  </parent>
+
+
+  <repositories>
+    <repository>
+      <id>googleapis</id>
+      <url>https://google-api-client-libraries.appspot.com/mavenrepo</url>
+    </repository>
+  </repositories>
+
+  <dependencies>
+    <dependency>
+        <groupId>com.google.apis</groupId>
+        <artifactId>google-api-services-cloudmonitoring</artifactId>
+        <version>v2beta2-rev20-1.20.0</version>
+    </dependency>
+    <dependency>
+        <groupId>com.google.oauth-client</groupId>
+        <artifactId>google-oauth-client</artifactId>
+        <version>${project.oauth.version}</version>
```

Comment:
I think it's better if we move this into the dependency management of the root POM to make sure we stay consistent on versions.